tests/crypto/data/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# OS-specific configuration
+if (UNIX)
+    set(BUILD_OPT --bash)
+elseif (WIN32)
+    set(BUILD_OPT --gitbash)
+else()
+    message(FATAL_ERROR "Unknown OS. Only supported OSes are Linux and Windows")
+endif()
+
+add_custom_target(
```

Comment:
Are you sure you want this instead of `add_custom_command`? The explicit listing of `BYPRODUCTS` makes me think you mean `OUTPUT`...

Reply:
The choice of using a custom target is intended so that the collection of collateral generated is a single node in the cmake graph for the host/enclave folders to take a dependency on.

The listing of byproducts is really just to enable potential optimization for ninja, they're not explicit targets.